app/build.gradle.kts

```diff
@@ -97,6 +97,7 @@ dependencies {
 
     // Others
     implementation(libs.kotlinx.serialization)
+    implementation(libs.androidx.core.splashscreen)
```

Comment:
In the end we need this library? Because last time we talked didn't we end up seeing that the library wasn't necessary and a single line with methods coming for the default android packages was enough?

Reply:
In kMail it's not used because we have a separated Activity to handle the SplashScreen.

In SwissTransfer, the MainActivity is handling the SplashScreen, so we need this library to be able to set the correct theme when the SplashScreen is done.